### PR TITLE
Update conclusion.md

### PR DESCRIPTION
**Pull request by gregrickaby**
- Netlify link redirects to https://clever-banach-415c03.netlify.app/
- Looks like the official docs have moved to https://storybook.js.org/docs/react/get-started/introduction
- And the Visual Testing Handbook 404's. I believe the new handbook is here (but is in progress): https://www.learnstorybook.com/visual-testing-handbook/

### CODE REVIEW

**Comment by domyen** [comment]
Hey Greg, thanks so much for the PR. One favor, would you do a global find and replace to swap the broken links out in our translations as well? 🙏

**Comment by gregrickaby** [comment]
@domyen Done! 👍🏻

File: content/intro-to-storybook/angular/en/conclusion.md [hunk 1]
```diff
@@ -7,19 +7,19 @@ Congratulations! You created your first UI in Storybook. Along the way you learn
...
+[🌎 **Deployed Storybook**](https://clever-banach-415c03.netlify.app/)
 
 Storybook is a powerful tool for React, React Native, Vue, Angular, Svelte and many others frameworks. It has a thriving developer community and a wealth of addons. This introduction scratches the surface of what’s possible. We’re confident that once you adopt Storybook, you’ll be impressed by how productive it is to build durable UIs.
 
 ## Learn more
 
 Want to dive deeper? Here are helpful resources.
 
-- [**Official Storybook documentation**](https://storybook.js.org/basics/introduction/) has API documentation, community links, and the addon gallery.
+- [**Official Storybook documentation**](https://storybook.js.org/docs/react/get-started/introduction) has API documentation, community links, and the addon gallery.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/angular/en/conclusion.md:18]: @gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/angular/get-started/introduction" in the "Official Storybook documentation" link.

File: content/intro-to-storybook/angular/es/conclusion.md [hunk 2]
```diff
@@ -7,19 +7,19 @@ description: 'Pon todo tu conocimiento junto y aprende más técnicas de Storybo
...
+[🌎 **Storybook Desplegado**](https://clever-banach-415c03.netlify.app/)
 
 Storybook es una poderosa herramienta para React, Vue y Angular. Cuenta con una próspera comunidad de desarrolladores y una gran cantidad de complementos. Esta introducción demuestra una pequeña parte de todo lo que es posible con Storybook. Estamos seguros de que una vez que adoptes Storybook, quedarás impresionado por lo productivo que es crear interfaces de usuario duraderas.
 
 ## Aprende más
 
 ¿Quieres profundizar tus conocimientos? Aquí encontrarás algunos recursos útiles:
 
-- [**Documentación oficial de Storybook**](https://storybook.js.org/basics/introduction/) contiene la documentación del API, links comunitarios y una galería de complementos.
+- [**Documentación oficial de Storybook**](https://storybook.js.org/docs/react/get-started/introduction) contiene la documentación del API, links comunitarios y una galería de complementos.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/angular/es/conclusion.md:18]: @gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/angular/get-started/introduction" in the "Official Storybook documentation" link.

File: content/intro-to-storybook/angular/pt/conclusion.md [hunk 3]
```diff
@@ -16,12 +16,12 @@ Possui uma comunidade de programadores próspera e uma grande variedade de extra
 
 Pretende saber mais? Aqui ficam alguns recursos que irão ajudar.
 
-- [**Documentação Oficial Storybook**](https://storybook.js.org/basics/introduction/) contém a documentação da API, links da comunidade e a galeria de extras.
+- [**Documentação Oficial Storybook**](https://storybook.js.org/docs/react/get-started/introduction) contém a documentação da API, links da comunidade e a galeria de extras.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/angular/pt/conclusion.md:19]: @gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/angular/get-started/introduction" in the "Official Storybook documentation" link.

File: content/intro-to-storybook/svelte/en/conclusion.md [hunk 4]
```diff
@@ -7,19 +7,19 @@ Congratulations! You created your first UI in Storybook. Along the way you learn
...
+[🌎 **Deployed Storybook**](https://clever-banach-415c03.netlify.app/)
 
 Storybook is a powerful tool for React, React Native, Vue, and Angular. It has a thriving developer community and a wealth of addons. This introduction scratches the surface of what’s possible. We’re confident that once you adopt Storybook, you’ll be impressed by how productive it is to build durable UIs.
 
 ## Learn more
 
 Want to dive deeper? Here are helpful resources.
 
-- [**Official Storybook documentation**](https://storybook.js.org/basics/introduction/) has API documentation, community links, and the addon gallery.
+- [**Official Storybook documentation**](https://storybook.js.org/docs/react/get-started/introduction) has API documentation, community links, and the addon gallery.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/svelte/en/conclusion.md:18]: @gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/svelte/get-started/introduction" in the "Official Storybook documentation" link.

File: content/intro-to-storybook/svelte/es/conclusion.md [hunk 5]
```diff
@@ -7,15 +7,15 @@ Felicitaciones! Creaste tu primer interfaz de usuario en Storybook. En el camino
...
+[🌎 **Deployed Storybook**](https://clever-banach-415c03.netlify.app/)
 
 Storybook es una poderosa herramienta para React, React Native, Vue, Angular, Svelte y muchos otros frameworks. Cuenta con una próspera comunidad de desarrolladores y una gran cantidad de complementos. Esta introducción nos lleva superficialmente a lo que es posible. Estamos seguros de que una vez que adoptes Storybook, quedarás impresionado por lo productivo que es crear interfaces de usuario duraderas.
 
 ## Aprende más
 
 Quieres bucear más profundo? Aquí algunos recursos útiles:
 
-- [**Documentación oficial de Storybook**](https://storybook.js.org/basics/introduction/) tiene la documentación del API, links comunitarios y una galería de complementos.
+- [**Documentación oficial de Storybook**](https://storybook.js.org/docs/react/get-started/introduction) tiene la documentación del API, links comunitarios y una galería de complementos.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/svelte/es/conclusion.md:18]: @gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/svelte/get-started/introduction" in the "Official Storybook documentation" link.

File: content/intro-to-storybook/vue/en/conclusion.md [hunk 6]
```diff
@@ -7,19 +7,19 @@ Congratulations! You created your first UI in Storybook. Along the way you learn
...
+[🌎 **Deployed Storybook**](https://clever-banach-415c03.netlify.app/)
 
 Storybook is a powerful tool for React, React Native, Vue, Angular, Svelte and many others frameworks. It has a thriving developer community and a wealth of addons. This introduction scratches the surface of what’s possible. We’re confident that once you adopt Storybook, you’ll be impressed by how productive it is to build durable UIs.
 
 ## Learn more
 
 Want to dive deeper? Here are helpful resources.
 
-- [**Official Storybook documentation**](https://storybook.js.org/basics/introduction/) has API documentation, community links, and the addon gallery.
+- [**Official Storybook documentation**](https://storybook.js.org/docs/react/get-started/introduction) has API documentation, community links, and the addon gallery.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/en/conclusion.md:18]: @gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/vue/get-started/introduction" in the "Official Storybook documentation" link.

File: content/intro-to-storybook/vue/pt/conclusion.md [hunk 7]
```diff
@@ -16,12 +16,12 @@ Possui uma comunidade de programadores próspera e uma grande variedade de extra
 
 Pretende saber mais? Aqui ficam alguns recursos que irão ajudar.
 
-- [**Documentação Oficial Storybook**](https://storybook.js.org/basics/introduction/) contém a documentação da API, links da comunidade e a galeria de extras.
+- [**Documentação Oficial Storybook**](https://storybook.js.org/docs/react/get-started/introduction) contém a documentação da API, links da comunidade e a galeria de extras.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/pt/conclusion.md:19]: @gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/vue/get-started/introduction" in the "Official Storybook documentation" link.

**Comment by gregrickaby** [comment]
@jonniebigodes I'm so sorry for the delay. I've updated the URL's.

**Comment by jonniebigodes** [comment]
@gregrickaby i saw the notification and since you've changed this pull request until now, there were some significant changes. The React tutorial was updated to 6.0 and was fined tuned with a follow up pull request. Also a japanese translation was added. So hence why the large number of files changed.

If you're ok we can address this with a smaller follow up pull request, focusing only your proposed changes, to avoid introducing back the same files once again. 

With that i would ask that you close this one and just make a new one with your proposed changes and add me as reviewer and we'll merge it as soon as it's up. I've checked the changes you made and we're good.

Sounds good?

Stay safe

**Pull request closed without merging**